Averages/projects/averages_project_2_4.py

```diff
@@ -98,9 +34,35 @@ def get_validated_input(question,input_type):
         print("Average Error: ", average_error)
         print("Total trials: ", tries)
   
-        
-        play = input("Want to play again? (y/n)\n")
+        print("Press Ctrl + C to stop at anytime. Moving on to the next round.")
 
+#calculate and print out the prediction
+def get_prediction(url, data={"A":48,"B":23,"C":38,"D":54}):
+    r = requests.post(url, data=json.dumps(data))
+    response = getattr(r,'_content').decode("utf-8")
+    response = json.loads(response)
+    prediction_object = json.loads(response['body'])
+    label = ""    
+    if 'predicted_label' in prediction_object:
+        label = prediction_object['predicted_label']
+
+    print("ML prediction") 
+    print("\tLabel: ", label)
+    print("\tModel: ", prediction_object['Model'])
+    # print("\tMessage: ", prediction_object['Message']) #UNCOMMENT WHEN MESSAGE KEY EXISTS FOR REGRESSION MODELS
```

Comment:
Is there a ticket for the regression models to add message?

Reply:
Not that I saw, but I think Sindhu said she needs to change one thing and the message will be there. I'll add a ticket just in case.

Reply:
Thanks!